### ISSUE TITLE
No "How to Guide" for macOS Sierra.

### ISSUE THREAD

**Issue by cRAN-cg** [michaelbromley/skqw#9]
![screen shot 2017-02-22 at 5 47 38 pm](https://cloud.githubusercontent.com/assets/8614844/23211219/0624a428-f927-11e6-9ade-7007163d09c9.png)

How do I configure my machine to run it ? [Is there any alternative to Setero Mix for mac ?]

**Comment by michaelbromley** [comment]
I don't have a mac and have no easy access to one, hence the lack of guides.

A quick search on Google seems to indicate there is no easy built-in way to use "stereo mix" as an audio input. Looks like a solution might be "soundflower": https://github.com/mattingalls/Soundflower/releases/tag/2.0b2

If you manage to get that working, feedback will be welcome. I can then update the docs with a successful approach.

**Comment by NathanWalker** [comment]
Looks like this is related to what I was seeing as on macOS Sierra I see the same thing, just the launch screen. I'll try Soundflower soon and report back.

**Comment by ThatIsAPseudo** [comment]
It doesn't work with Soundflower, i still have the launch screen and nothing else.

**Comment by cbmtrx** [comment]
Yeah unfortunately on OS Catalina I don't get anything beyond the grey startup screen.

**Comment by michaelbromley** [comment]
Sorry everyone. I had to abandon this due to being totally unable to test on MacOS (plus having kids 😆 ). 

If I suddenly got a lot more free time I'd love to have a go at re-building this from the ground up with https://tauri.app/

**Comment by cbmtrx** [comment]
Yeah kids will do that to ya 🤪

Been using Chromata which is great fun.
